platform/ui/src/views/StudyList/components/StudyListFilter.js

```diff
@@ -275,12 +182,32 @@ const StudyListFilter = ({
   );
 };
 
+StudyListFilter.defaultProps = {
+  filtersMeta: [],
+  filtersValues: {
+    patientName: '',
+    mrn: '',
+    studyDate: {
+      startDate: null,
+      endDate: null,
+    },
+    description: '',
+    modality: [],
+    accession: '',
+    sortBy: '',
+    sortDirection: 0,
+    page: 0,
+    resultsPerPage: 25,
+  },
+  numOfStudies: 90,
+};
```

Comment:
On the fence about defaults here. Less pressing as this is higher up; might be worth forcing above layer to provide values so its easier to see at a glance where everything is coming from.

We should have a dedicated page for just `StudyListFilter`